src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/pagination/RestResponsePage.kt

```diff
@@ -0,0 +1,28 @@
+package uk.gov.justice.digital.hmpps.hmppsmanageadjudicationsapi.pagination
+
+import com.fasterxml.jackson.annotation.JsonCreator
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.fasterxml.jackson.databind.JsonNode
+import org.springframework.data.domain.PageImpl
+import org.springframework.data.domain.PageRequest
+import org.springframework.data.domain.Pageable
+
+class RestResponsePage<T> : PageImpl<T> {
```

Comment:
Why do we need this? It seems to just pass everything thru to PageImpl?

Reply:
Yeah - it does - but the Spring instance (PageImpl) can't be deserialised by jackson as there is no default constructor on it - hence the need for this :(